lib/qa/authorities/web_service_base.rb

```diff
@@ -32,6 +32,7 @@ def get_json(url)
     # @param url [String]
     # @return [Faraday::Response]
     def response(url)
+      Faraday::Utils.default_params_encoder = Faraday::FlatParamsEncoder
```

Comment:
Yeah, that patch does fix it. Sorry!  https://github.com/projecthydra-labs/questioning_authority/blob/43cba871eda61b55096321e10e486254762f939e/lib/qa/authorities/loc/generic_authority.rb